### PR TITLE
GDG Watch Face

### PR DESCRIPTION
**Pull request by jomartigcal**
Adding the current version of the GDG Watch Face

Signed-off-by: Jomar Tigcal <jomar@tigcal.com>

### CODE REVIEW

File: wear/build.gradle [hunk 1]
```diff
@@ -0,0 +1,27 @@
+apply plugin: 'com.android.application'
+
+
+android {
+    compileSdkVersion 23
```
Comment (tasomaniac) [inline wear/build.gradle]: Can you retrieve these from root projects Gradle file

Comment (jomartigcal) [inline wear/build.gradle]: Sorry about that. Will update.

File: wear/build.gradle [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    buildToolsVersion "23.0.2"
+
+    defaultConfig {
+        applicationId "org.gdg.frisbee.android"
+        minSdkVersion 21
+        targetSdkVersion 23
```
Comment (tasomaniac) [inline wear/build.gradle]: Our target version is 22. So lets make the same target version here as well.

File: wear/build.gradle [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            minifyEnabled false
+            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+        }
+    }
+}
+
+dependencies {
+    compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile 'com.google.android.support:wearable:1.3.0'
+    compile 'com.google.android.gms:play-services-wearable:8.4.0'
```
Comment (tasomaniac) [inline wear/build.gradle]: I guess we have to use play services when writing Watch Faces right? Can you use the version we use in root build.gradle. Thank you :)

**Comment by friedger** [comment]
Nice! Could you use the new logo without rounded corners?
Please see https://developers.google.com/groups/logo/

File: wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+                    } else {
+                        settingTextView.setText(getString(R.string.label_setting_on));
+                        circleImage.setImageResource(R.drawable.ic_date_on);
+                        putDataMapRequest.getDataMap().putBoolean(action, true);
+                        mDisplayDate = true;
+                        updateConfigurations();
+                    }
+
+                    PutDataRequest putDataRequest = putDataMapRequest.asPutDataRequest();
+                    Wearable.DataApi.putDataItem(mGoogleApiClient, putDataRequest);
```
Comment (tasomaniac) [inline wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java]: Data API is for syncing the data between phone and watch, right? If we don't have phone counterpart for config screen, should we remove these? Do we have settings page in the phone for the watch face?

**Comment by tasomaniac** [comment]
Thank you for the contribution. Apart from the inline comments, I have 1 more comment: Do we need ic_launcher.png assets? If not, can you remove them. Thank you.

**Comment by jomartigcal** [comment]
Sorry about some of the bugs. I'll recheck and push updates.

**Comment by tasomaniac** [comment]
It's totally fine. Thank you for the contribution. Waiting for the next commit.

File: wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java (hunk 4 above)
Comment (jomartigcal) [inline wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java]: This was an old code. I'll refactor and test.

**Comment by jomartigcal** [comment]
Ah! I forgot there was an ic_launcher-web.

File: wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java (hunk 4 above)
Comment (jomartigcal) [inline wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java]: The plan is to add mobile config screen

Comment (tasomaniac) [inline wear/src/main/java/org/gdg/frisbee/android/WearableConfigurationActivity.java]: I think that they should be added when the mobile config screen is created. But if you don't have time to refactor, they can stay.

**Comment by friedger** [comment]
Thank you for your contribution!

**Comment by tasomaniac** [comment]
Let's ship this

**Comment by tasomaniac** [comment]
@jomartigcal Did you test the release version of the APK? I created a release version. I checked that micro-apk is included in the file. But somehow it does not install the watchface to my watch.

**Comment by jomartigcal** [comment]
How did you test the release version? The automatic install of watchface APK to the wear device only happens when installed from Google Play Store.

**Comment by tasomaniac** [comment]
Are you sure? As far as I know, if the release APK with micro wear apk in it is installed, that would install it to the watch.

**Pull request merged**